packages/shared/src/components/cards/ShowMoreContent.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement, ReactNode, useState } from 'react';
+import { ClickableText } from '../buttons/ClickableText';
+
+interface ShowMoreContentProps {
+  content: string;
+  charactersLimit?: number;
+  contentPrefix?: ReactNode;
+  className?: string;
+}
+
+export default function ShowMoreContent({
+  content,
+  charactersLimit = 150,
+  contentPrefix,
+  className,
+}: ShowMoreContentProps): ReactElement {
+  const [isTextExpanded, setIsTextExpanded] = useState(false);
+  const linkName = isTextExpanded ? 'Show less' : 'Show more';
+
+  const toggleTextExpanded = () => {
+    setIsTextExpanded(!isTextExpanded);
+  };
+
+  const displayShowMoreLink = () => {
+    if (content?.length > charactersLimit) {
+      return linkName;
+    }
+    return '';
+  };
+
+  const getSlicedContent = () => {
+    return content?.slice(0, charactersLimit);
+  };
+  return (
+    <div className={className}>
+      <p className="typo-callout">
```

Comment:
I think this should be `typo-headline` looking at the spec.

![Screenshot 2022-01-19 at 09 52 23](https://user-images.githubusercontent.com/554874/150087386-456a9123-789d-4376-baf0-9ba66d144295.png)

Reply:
I applied `typo-headline` before but with that all text becomes bold:

![Screenshot 2022-01-19 at 13 49 23](https://user-images.githubusercontent.com/33021204/150134113-c320f48a-c7e6-42d4-b072-a1a39985eb1d.png)

Reply:
Hmm, you might have to double check with Tsahi.
I think it should be 17px, but maybe not bold.

Reply:
There is a bold font-weight in typo-headline:
```
 '.typo-headline': {
      'font-size': '1.0625rem',
      'line-height': '1.5rem',
      'font-weight': 'bold',
    }
```

Reply:
Clarified with @tsahimatsliah, it should be typo-body.